courseraresearchexports/commands/jobs.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Coursera
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from courseraresearchexports.exports.constants import SCHEMA_NAMES, \
+    EXPORT_TYPES, ANONYMITY_LEVELS
+from courseraresearchexports.exports import api
+from courseraresearchexports.exports import utils
+from datetime import datetime
+import json
+import argparse
+
+
+def download(args):
+    """
+    Download research export job using a job id.
+    """
+    job = api.get(args.id)
+    utils.download_export(job, args.dest)
+
+
+def create(args):
+    """
+    Creates a post request to Coursera's research exports service
+     with specified parameters
+    """
+    export_job_json = utils.create_export_job_json(**vars(args))
+    export_job_id = api.create(export_job_json)
+
+    print('Successfully created job {}:\n{}'.format(
+        export_job_id,
+        json.dumps(export_job_json, indent=2)))
+
+
+def get(args):
+    """
+    Get status of a research export job using a job id.
+    """
+    job = api.get(args.id)
+
+    scope_id, scope_name = utils.get_scope_id_and_name_from_job(job)
+
+    export_type = job['exportType']
+
+    if export_type == 'RESEARCH_WITH_SCHEMAS':
```

Comment:
can you factor 'RESEARCH_WITH_SCHEMAS' into a constant so if we change this string on the BE, we only need to change it in 1 place?